pom.xml

```diff
@@ -46,6 +46,38 @@
             <type>pom</type>
         </dependency>
 
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.6.1</version>
```

Comment:
please choose the latest verion
http://mvnrepository.com/artifact/org.slf4j/slf4j-log4j12/1.7.12